common/utils.js

```diff
@@ -116,6 +118,27 @@ export class Encoding {
         return Encoding.makeWebsafe(shortKey.toString('base64'));
     }
 
+    /**
+     * Encodes element key to system id
+     * @param {string} key 
+     * @returns {string}
+     */
+    static toSystemId(fullKey) {
```

Comment:
This code seem to appear in other PR, but I guess it's just to have an independent PRs.

Reply:
Yep.